### ISSUE TITLE
[SPIKE] Can or how can we support creating grouped pre-check-in appts in our stg tool? 

### ISSUE THREAD

**Issue by benbrasso-agile6** [department-of-veterans-affairs/va.gov-team#64359]
## Description 

How can we create the following scenarios in our appt scheduling stg tool? 

- Two in-person appts on same day (same VistA)
- One in-person appt, one phone appt on same day (same VistA)

Note: I think this will be nice to have as we move forward with unified check in. Of course, I don't know the % of single appts vs. >1 appts for pre-check-ins. We would need to ask Shawn or Shane for some % if we wanted to know whether this is valuable work or not. 

## AC

- [ ] Provide decision and/or create ticket for next step

**Comment by loripusey** [comment]
@benbrasso-agile6 ; @brianseek says you can already do this, can you test?

**Comment by benbrasso-agile6** [comment]
How? I know I can set two appointments on the same day. But, for reviewing them in pre-check-in, how do I "group" them so I'm completing pre-check-in for both appts?

**Comment by loripusey** [comment]
@acrollet would it be possible for you to type up a summary of how the grouped appointments work? I think the current engineers are not that familiar with the functionality and we have no way to test it or see it in action. I'd like to get the team on the same page with respect to our knowledge of this feature and how it works.  Thanks.
cc: @benbrasso-agile6 @brianseek

**Comment by brianseek** [comment]
@benbrasso-agile6 yeah you are right, sorry about that. I got my wires crossed with day-of. I think I can set this up manually for you. Or you can run the application locally to see it. Find a time on my cal and we can co-work it.

**Comment by benbrasso-agile6** [comment]
Cool, but do we have enough info to point the implementation to add it into the staging tool? So, we don't have to rely on you and/or local set up?

For example, thinking of solutioning, the modal that appears to grab the UUID, we could add another UUID below it that fetches any appts available for pre-check-in on that same day (to simplify things). Again, solutioning now probably isn't appropriate.

**Comment by brianseek** [comment]
I think we would need to discuss if this is worth the effort to add into the tool. Is this something you see your self needing to set up often? Pre-check-in is going to be changing on the CHIP side in the near future, so I hesitate to prioritize this functionality unless its a heavy use feature.

**Comment by benbrasso-agile6** [comment]
Will let Lori figure this one out @loripusey. I reached out to Shawn to see if he has info about frequency of grouped pre-check-in sessions.

**Comment by loripusey** [comment]
Until we are finished with the ATO work, I don't think we have extra capacity to look at this one. We'll make a decision when we find out frequency.

**Comment by benbrasso-agile6** [comment]
Got some info from Shawn, but I'm going to pass it to Patrick Bateman to run some CDW queries

**Comment by benbrasso-agile6** [comment]
Sent message to patrick, told him it wasn't urgent

**Comment by benbrasso-agile6** [comment]
Patrick turned me to Shane. Sent message to Shane, looks like he's OOO. But, will wait and see.

**Comment by acrollet** [comment]
re: how grouped appointments work. Digging into the vetext source a bit, for appointments to be grouped by vetext, they have to be in clinics that are marked as part of a group: https://github.com/department-of-veterans-affairs/vetext/blob/951581e09c509d5cd1ec566ca097b5fbcb187044/ll-vetext-service/src/main/java/gov/va/med/lom/vetext/tasks/workers/impl/AppointmentRemindersWorkerImpl.java#L736

We would need to ask @robertdurkin for info about how clinic groupings are stored, I suspect in VEText configuration but it might be in VistA. To do this ourselves we would need to set up grouped clinics in 500/530.

**Comment by benbrasso-agile6** [comment]
Didn't get too far with this, so staying in the backlog

> Sorry for the delay, I was on Leave last week.  VEText stores the grouped information in the VEText Database.  There are 181 active clinic groupings. with various numbers of clincIens separated by'^' in one field.   I have attached the data for you here.

[Uploading... grouping (1).xls]
